webmails/snappymail/Dockerfile

```diff
@@ -74,7 +66,7 @@ RUN cd /var/www/webmail \
 COPY login/include.php /var/www/webmail/include.php
 COPY login/sso.php /var/www/webmail/sso.php
 
-COPY start.py /start.py
+COPY start.py /
```

Comment:
Elsewhere we specify the full path for the destination

Reply:
That was for a temporary test. I forgot to change it back.

Reply:
I changed it everywhere to the folder only in the base PR. So IMHO `COPY start.py /` is fine for me.

Reply:
I will make sure it is consistent as nextgens mentioned in the chat